gauges/vacuum.go

```diff
@@ -94,3 +96,25 @@ func (g *Gauges) LastTimeAutoVacuumRan() *prometheus.GaugeVec {
 
 	return gauge
 }
+
+// VacuumRunningTotal returns the number of backends (including autovacuum worker processes)
+// that is currently running a vacuuming (not include VACUUM FULL).
+//
+// This metric is only supported for PostgreSQL 9.6 or newer versions
+func (g *Gauges) VacuumRunningTotal() prometheus.Gauge {
+	var gaugeOpts = prometheus.GaugeOpts{
+		Name:        "postgresql_vacuum_running_total",
+		Help:        "Number of backends (including autovacuum worker processes) that is currently vacuuming",
+		ConstLabels: g.labels,
+	}
+
+	const vacuumRunningQuery = `
+		SELECT COUNT(*) FROM pg_stat_progress_vacuum WHERE datname = current_database()
+	`
+
+	if !postgres.Version(g.version()).Is96Or10() {
```

Comment:
should we change this to `Is96OrAbove`?

Reply:
Yes! For this we need to change the version function because it use the string format instead of the numeric one. 

I’ll change this in a specific PR